### PR TITLE
Feature/specification headers 117833165

### PR DESCRIPTION
**Pull request by shavkat-samatov**
https://www.pivotaltracker.com/story/show/117833165

### CODE REVIEW

**Comment by ethansr** [comment]
:+1:

File: lib/gap_intelligence/client/headers.rb [hunk 1]
```diff
@@ -36,5 +36,17 @@ def pricing_headers(params = {}, options = {}, &block)
       default_option(options, :record_class, Header)
       perform_request(:get, 'headers/pricings', options.merge(params: params), &block)
     end
+
+    # Requests a list of pricing headers
```
Comment (gtimti) [inline lib/gap_intelligence/client/headers.rb]: The line says `pricing headers`, should be `specification headers`.

File: spec/factories/headers.rb [hunk 2]
```diff
@@ -33,7 +31,23 @@
...
+                            'position' => nil }
+        }
+      }
+    end
+
+    trait :specifications do
+      initialize_with {
+        {
+          'id' => '',
+          'type' => 'pricing_headers',
```
Comment (gtimti) [inline spec/factories/headers.rb]: Wrong type, should be `specification_headers`

**Comment by gtimti** [comment]
looks good to me. 👍

**Comment by shavkat-samatov** [comment]
@gtimti fixed typos, thanks! 👓

**Pull request merged**